src/main/groovy/edu/oregonstate/mist/osuevents/resources/EventsResource.groovy

```diff
@@ -221,7 +199,75 @@ class EventsResource extends Resource {
         eventsDAO.deleteEvent(id)
         Response.noContent().build()
     }
+/**
+ * Helper function for data validation. Returns list of applicable errors.
+ */
+    private List<Error> getErrors(ResultObject resultObject,
+                                  String pathID = null) {
+        List<Error> errors = []
+        ResourceObject resourceObject
+        Event event
+
+        if (pathID) {
+            if (!eventsDAO.getById(pathID)) {
+                errors.add(new Error(
```

Comment:
The skeleton should have a method that can returns a notFound error object. There's the notFound() method but that returns a ResponseBuilder object. Can you create a separate ticket to make that enhancement and update this to just use the skeleton?